eth2near/eth2near-block-relay-rs/src/hand_made_finality_light_client_update.rs

```diff
@@ -233,6 +290,57 @@ mod tests {
             serde_json::to_string(&hand_made_light_client_update.sync_aggregate).unwrap(),
             serde_json::to_string(&light_client_update.sync_aggregate).unwrap()
         );
+    }
+
+    #[ignore]
+    #[test]
+    fn test_hand_made_finality_light_client_update() {
+        let beacon_rpc_client = BeaconRPCClient::new(BEACON_ENDPOINT);
+        let hand_made_light_client_update =
+            HandMadeFinalityLightClientUpdate::get_finality_light_client_update(
+                &beacon_rpc_client,
+                ATTESTED_SLOT,
+                true,
+            )
+            .unwrap();
+        let light_client_update = beacon_rpc_client.get_light_client_update(99).unwrap();
```

Comment:
the used number should be a constant

Reply:
fix: https://github.com/aurora-is-near/rainbow-bridge/pull/787/commits/4ac0124069e6fd0b34cee92e150620159c96454b